CMakeLists.txt

```diff
@@ -134,6 +134,10 @@ find_package (Eigen3 3.3 REQUIRED NO_MODULE)
 ### XLNT
 find_package (XLNT REQUIRED)
 
+### Libigl set include path
+include_directories("${CMAKE_PREFIX_PATH}/../build/libigl/include/")
+#MESSAGE("THIS ISNISANOSBCDOSCF ${CMAKE_PREFIX_PATH}")
```

Comment:
Let's remove this line.

Reply:
Wont that cause the headers like ```<igl/cot.h>``` fail and we will have to use the whole path ?

Yeah I dont think we need the find package for this lib.

Reply:
I just meant the commented out message line.

Reply:
oh ok